[2025-05-09T16:04:28.258+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.update_hql_file manual__2025-05-09T16:04:25.744799+00:00 [queued]>
[2025-05-09T16:04:28.268+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.update_hql_file manual__2025-05-09T16:04:25.744799+00:00 [queued]>
[2025-05-09T16:04:28.269+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-09T16:04:28.283+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): update_hql_file> on 2025-05-09 16:04:25.744799+00:00
[2025-05-09T16:04:28.290+0000] {standard_task_runner.py:57} INFO - Started process 2095 to run task
[2025-05-09T16:04:28.292+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_transaction_data_and_upload_to_hdfs', 'update_hql_file', 'manual__2025-05-09T16:04:25.744799+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmpnyl0q5qq']
[2025-05-09T16:04:28.296+0000] {standard_task_runner.py:85} INFO - Job 39: Subtask update_hql_file
[2025-05-09T16:04:28.350+0000] {task_command.py:415} INFO - Running <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.update_hql_file manual__2025-05-09T16:04:25.744799+00:00 [running]> on host d4afc872a8aa
[2025-05-09T16:04:28.446+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_transaction_data_and_upload_to_hdfs' AIRFLOW_CTX_TASK_ID='update_hql_file' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T16:04:25.744799+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-09T16:04:25.744799+00:00'
[2025-05-09T16:04:28.453+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/context.py:314 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-05-09T16:04:28.607+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-09T16:04:28.630+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_transaction_data_and_upload_to_hdfs, task_id=update_hql_file, execution_date=20250509T160425, start_date=20250509T160428, end_date=20250509T160428
[2025-05-09T16:04:28.668+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-09T16:04:28.694+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
